README.md

```diff
@@ -161,3 +164,11 @@ Client contracts which initiate a request and receive a fulfillment can be modif
 ## Simulating Requests
 
 An end-to-end request initiation and fulfillment can be simulated using the `npx hardhat functions-simulate` command. This command will report the total estimated cost of a request in LINK using the latest on-chain gas prices. Costs are based on the amount of gas used to validate the response and call the client contract's `fulfillRequest` function, plus a flat fee. Please note that actual request costs can vary based on gas prices when a request is initiated on-chain.
+
+## Off-chain Secrets
+
+Instead of using encrypted secrets stored directly on the blockchain, encrypted secrets can also be hosted on URLs and be fetched by the DON nodes when a request is initiated.  To generate the encrypted secrets JSON file, run the command `npx hardhat functions-build-offchain-secrets --network network_name_here`.  This will output the file `offchain-secrets.json` which can be hosted at any URL such as Github or S3.
```

Comment:
Files are not "hosted on URLs", URLs point to hosting locations. Reword to just "hosted off-chain" and then at the end "hosted anywhere (e.g. Github, S3, etc)."

Also add that currently HTTP is the only supported protocol.

Reply:
Improved wording